### DISCUSSION TITLE
License file location

### DISCUSSION THREAD

**Discussion by mitchelltrehernevet** [aclap-dev/video-downloadhelper#1040] [Help and Q&A]
### Your question

Hi. I recently reset my pc and I backed stuff up so I am pretty sure I have my license somewhere. Does anyone know the location it usually gets saved to? And what the filename might be?  Thank you.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 2
}

**Comment by jcfrog** [comment]
Hi,
Did you try here: https://www.downloadhelper.net/recover-license

**Reply by mitchelltrehernevet** [reply]
I did, yes. Thank you.

**Reply by mitchelltrehernevet** [reply]
I also sent an email to support@downloadhelper.net
